XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/client/WSCOORClient.java

```diff
@@ -41,12 +42,27 @@
     private static synchronized ActivationService getActivationService()
     {
         if (activationService.get() == null) {
-            // activationService.set(new ActivationService(null, new QName("http://docs.oasis-open.org/ws-tx/wscoor/2006/06", "ActivationService")));
-            activationService.set(new ActivationService());
+            setActivationService();
```

Comment:
I have a query about the style you used throughout this patch. I see you added "private static synchronized" methods throughout and called them from places where you replaced the original setter. I wondered if it would read better if the calls were still inline. Is there a security style/precedent that you were following or was it personal preference?